### ISSUE TITLE
Collateral

### ISSUE THREAD

**Issue by wartycauldron** [Berry-Pool/nami-wallet#334]
If ever I reset my nami wallet? Does it automatically remove the collateral?

**Comment by alessandrokonrad** [comment]
Yes collateral is always part of your balance actually. It's only seperated in the current Nami instance you have. Whenever you reimport the wallet or use another one, you see the whole balance again.
